src/index.js

```diff
@@ -1,26 +1,8 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './Style/Main.css';
-import App from './Component/App';
-import registerServiceWorker from './ServiceWorker';
```

Comment:
shouldn't this file be removed if not use anymore ?

Reply:
I think it's removed no ?

Reply:
oops my bad :p